src/routes/auth/auth.controller.js

```diff
@@ -1,6 +1,3 @@
-/**
```

Comment:
Yes, there are some really weird things that happened the way I had it set up, so I had to put it back, then that broke the jsdoc comments, so I removed them until I can get it figured out. However, since we won't ever be calling those functions explicitly, I decided it didn't matter much.